### ISSUE TITLE
cannot import batch normalization

### ISSUE THREAD

**Issue by lakpa-tamang9** [Dene33/video_to_bvh#40]
![image](https://user-images.githubusercontent.com/54692758/150664823-f5f4033e-b8aa-458b-8735-abf22944c9b3.png)

**Comment by 1783883121** [comment]
需要加上
!pip uninstall keras tensorflow
!pip install keras==2.3.1
!pip install tensorflow==2.0

**Comment by jaygshah** [comment]
Or just change 
`from keras.layers.normalization import BatchNormalization` 
to
`from keras.layers import BatchNormalization`
in model_load.py
